pontoon/pretranslation/pretranslate.py

```diff
@@ -57,6 +57,15 @@ def get_pretranslations(entity, locale, preserve_placeables=False):
             return []
 
         pretranslation = serializer.serialize_entry(entry)
+        print(pretranslation)
+        # error-not-subscribed = Esta dirección de correo electrónico no está suscrita a {-product-name}.
+
+        # Parse and serialize pretranslation again in order to assure cannonical style
+        parsed_pretranslation = parser.parse_entry(pretranslation)
+        pretranslation = serializer.serialize_entry(parsed_pretranslation)
+        # error-not-subscribed = Esta dirección de correo electrónico no está suscrita a { -product-name }.
```

Comment:
Leaving actual translations in as comments just for the review purpose (I'll remove that before mering).

@eemeli Is there a more elegant way to achieve the same results?

Reply:
Not at the moment, as the results of the pretranslation are not properly validated: https://github.com/mozilla/pontoon/blob/c63b0c5f6526a4e3507d4e4e7af1463522ff06d3/pontoon/pretranslation/transformer.py#L239-L246